From 87eefa57c5f8f059df28788eb07fb7d6d9775d2d Mon Sep 17 00:00:00 2001
From: Yongxin Liu <yongxin.liu@windriver.com>
Date: Thu, 13 Jun 2024 10:26:39 +0000
Subject: [PATCH 12/24] rdk/snr: klm/cpk/ice_devlink.c: remove
 devlink_reload_disable()

devlink_reload_disable() was removed in kernel commit 82465bec3e97
("devlink: Delete reload enable/disable interface").

Upstream-Status: Inappropriate [RDK is released in tarball only]

Signed-off-by: Yongxin Liu <yongxin.liu@windriver.com>
---
 klm/cpk/ice_devlink.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/klm/cpk/ice_devlink.c b/klm/cpk/ice_devlink.c
index 510e32c..9a5399d 100644
--- a/klm/cpk/ice_devlink.c
+++ b/klm/cpk/ice_devlink.c
@@ -69,10 +69,6 @@ void ice_devlink_unregister(struct ice_pf *pf)
 {
 	struct devlink *devlink = priv_to_devlink(pf);
 
-#ifdef HAVE_DEVLINK_RELOAD_ACTION_AND_LIMIT
-	devlink_reload_disable(devlink);
-#endif /* HAVE_DEVLINK_RELOAD_ACTION_AND_LIMIT */
-
 #ifdef HAVE_DEVLINK_PARAMS
 	devlink_params_unregister(devlink, ice_devlink_params,
 				  ARRAY_SIZE(ice_devlink_params));
-- 
2.39.3

